editor/src/components/documentation/documentation-components.tsx

```diff
@@ -70,13 +70,10 @@ export const PrettyKeys = (props: PrettyKeysProps) => {
   return (
     <span>
       {[...props.shortcut].map((keyb, index) => (
-        <span key={(keyb === " ") ? `space-${index}`: keyb} style={{ ...keyStyle }}>
+        <span key={keyb === ' ' ? `space-${index}` : keyb} style={{ ...keyStyle }}>
```

Comment:
For anyone else that wonders why this change is in here, it looks to have been just a result of Prettier running on the file, likely as a result of latest master being merged into this branch.